src/Task/Recurrence.ts

```diff
@@ -261,11 +261,11 @@ export class Recurrence {
         options.dtstart = after.startOf('day').toDate();
         rrule = new RRule(options);
 
-        return window.moment(rrule.after(after.toDate()));
+        return window.moment.utc(rrule.after(after.toDate()));
     }
 
     private static addTimezone(date: Moment): Moment {
-        const localTimeZone = window.moment.utc(date).local(true);
+        const localTimeZone = window.moment.utc(date).add(12, 'hours').local(true);
```

Comment:
Does this depend on the date being at midnight?
That's fine for now, but I am asking as at some I am likely add support for times on Tasks dates.

For future maintainers, a comment explaining why the 'add 12 hours', and any assumptions behind it, will be really helpful.

Reply:
Indeed it did depend on `date` being midnight at UTC. I have now changed this to `.set({hour:12,minute:0,second:0,millisecond:0})` so that this wouldn't depend on `date` being midnight.

I have also added a comment describing the problem and the workaround.

Support for times on Tasks dates would be amazing! Obviously this workaround would need to be more complicated to work with times.